src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -45,32 +40,45 @@ import { isPlatformBrowser } from '@angular/common';
                 </ng-container>
             </ng-container>
             <ng-container *ngIf="tabsService.overflowTabs.length > 0">
-                <div class="tabs-overflow bottom-right" [class.open]="toggleService.open" role="presentation">
-                    <li role="application" class="nav-item" (click)="toggleOverflow($event)">
-                        <button class="btn btn-link nav-link dropdown-toggle" type="button" aria-hidden="true"
-                                [class.active]="activeTabInOverflow" [class.open]="inOverflow()" tabIndex="-1">
+                <div class="tabs-overflow bottom-right" role="presentation" 
+                  [class.open]="toggleService.open">
+                    <li role="application" class="nav-item">
+                        <button #tabOverflowTrigger class="btn btn-link nav-link dropdown-toggle" type="button" aria-hidden="true" 
+                        [attr.tabindex]="activeTabInOverflow && !toggleService.open? 0: -1"
+                        [class.active]="activeTabInOverflow"
+                        [class.open]="toggleService.open"
+                        (mousedown)="_mousedown = true"
+                        (focus)="openOverflowOnFocus()"
+                        (click)="toggleOverflowOnClick()">
                             <clr-icon shape="ellipsis-horizontal"
                               [class.is-info]="toggleService.open"
                               [attr.title]="commonStrings.keys.more"></clr-icon>
                         </button>
                     </li>
                     <!--tab links in overflow menu-->
-                    <clr-tab-overflow-content>
+                    <div class="dropdown-menu" #tabOverflow *ngIf="toggleService.open"  
```

Comment:
Not part of this, but we ought to move to smart popovers. 

This gets more interesting then because the popovers are in the body instead of inside the list.

Reply:
There were some talks to deprecate the tabs overflow in the long run. Did we reach a decision in favor/against it? If we still may want to do it, maybe we don't want the smart popovers transition here.

Reply:
No plans currently to change overflow.